google-cloud-core-grpc/src/main/java/com/google/cloud/grpc/BaseGrpcServiceException.java

```diff
@@ -72,8 +72,8 @@ public BaseGrpcServiceException(ApiException apiException) {
         .setMessage(apiException.getMessage())
         .setCause(apiException)
         .setRetryable(apiException.isRetryable())
-        .setCode(((GrpcStatusCode) apiException.getStatusCode()).getCode().value())
-        .setReason(((GrpcStatusCode) apiException.getStatusCode()).getCode().name())
+        .setCode(apiException.getStatusCode().getCode().getHttpStatusCode())
```

Comment:
I might be missing something. Why is this HTTP code? This is for gRPC right?

Reply:
Yes, but the base class `BaseServiceException` expects an integer code. It was previously using the proto field number for grpc exceptions, but I don't think that was useful to anyone.